src/styles/variables.ts

```diff
@@ -148,6 +148,7 @@ export default {
     signInContentMaxWidth: 1360,
     signInHeroContextMaxWidth: 680,
     signInContentMinHeight: 800,
+    signInSAMLContentMinHeight: 1000,
```

Comment:
What exactly does this do? Rather than setting a minHeight on this, can't we just fix the animated image itself so that it doesn't get cut off? 

I'm just thinking that there should be a way for the animated image to be a block-like element that gets wrapped in the overall content wrapper, without needing to set an explicit min height to account for it.

Reply:
Agree with @shawnborton here!

Reply:
Yes makes sense, but as far I can see, this was created to use a fixed height due to a bug with the footer in mobile screens, I'm afraid of if we use flexbox or another css technique we going to cause a regression, please see these commit where @grgia was trying to fix this issue:

- https://github.com/Expensify/App/commit/606d41d57d9808e78b4a7441c93aba1bce66a7cd
- https://github.com/Expensify/App/commit/f4f498cbe7b03d643714b3aff9c1fa26a660e03d
- https://github.com/Expensify/App/commit/fe6156652ab679671571b4b01c03d09fd42fafab
- https://github.com/Expensify/App/commit/12c947efea7fdad00bec3466f72139cc360d663d
- https://github.com/Expensify/App/commit/4c0f3070a9a997fb7a16d497d7ccde149e21d679
- etc

@allroundexperts, I suggest to solve this with the fixed height and in other PR try to fix the CSS issue, please confirm if you agree or if I should try to fix that in this PR.